### ISSUE TITLE
PyQt GUI for MDANSE

### ISSUE THREAD

**Issue by MBartkowiakSTFC** [ISISNeutronMuon/MDANSE#150]
As far as I know, we are planning to replace the current wxWidgets GUI with a Qt-based GUI sooner or later.

If everyone agrees, I could volunteer to write a new GUI using PyQt6. This would be tied to the protos branch.

Please give me some comments about the new GUI:
1. Do we need it?
2. Should it be a replica of the existing one, or should it be laid out differently?
3. Compared to the current GUI, are there any features that we want to drop or add?
4. Any other comments you find relevant.

**Comment by eurydice76** [comment]
Hi @Ratislav and @MBartkowiakSTFC , I am happy that what I recommended is finally on the way to be adopted namely the replacement of wxPython by PyQt. This was necessary because wxPython (aka phoenix) is not supported as much as PyQt especially I found some problem installing it via pip. Moreover, we also found a major problem with the vtk/wxPython binding on Windows which triggered a nasty segfault without any trace. Being the author of the wxPython current interface, I am not the right person to give an opinion, I let @gonzalezma and @sanghamitra-mukhopadhyay give their opinion about it.
